care/abdm/service/gateway.py

```diff
@@ -0,0 +1,214 @@
+                            "%Y-%m-%dT%H:%M:%S.000Z"
+                        ),
+                    },
+                    "dataEraseAt": consent.expiry.astimezone(timezone.utc).strftime(
+                        "%Y-%m-%dT%H:%M:%S.000Z"
+                    ),
+                    "frequency": {
+                        "unit": consent.frequency_unit,
+                        "value": consent.frequency_value,
+                        "repeats": consent.frequency_repeats,
+                    },
+                },
+            },
+        }
+
+        path = "/v0.5/consent-requests/init"
+        return self.request.post(path, data, headers={"X-CM-ID": settings.X_CM_ID})
+
+    def consent_requests__status(self, consent_request_id: str):
+        data = {
+            "requestId": str(uuid.uuid4()),
+            "timestamp": datetime.now(tz=timezone.utc).strftime(
+                "%Y-%m-%dT%H:%M:%S.000Z"
+            ),
+            "consentRequestId": consent_request_id,
+        }
+
+        return self.request.post(
+            "/v0.5/consent-requests/status", data, headers={"X-CM-ID": settings.X_CM_ID}
+        )
+
+    def consents__hiu__on_notify(self, consent: ConsentRequest, request_id: str):
+        data = {
+            "requestId": str(uuid.uuid4()),
+            "timestamp": datetime.now(tz=timezone.utc).strftime(
+                "%Y-%m-%dT%H:%M:%S.000Z"
+            ),
+            "resp": {"requestId": request_id},
+        }
+
+        if len(consent.consent_artefacts.all()):
+            data["acknowledgement"] = []
+
+        for aretefact in consent.consent_artefacts.all():
+            data["acknowledgement"].append(
+                {
+                    "consentId": str(aretefact.artefact_id),
+                    "status": "OK",
+                }
+            )
+
+        return self.request.post(
+            "/v0.5/consents/hiu/on-notify", data, headers={"X-CM-ID": settings.X_CM_ID}
+        )
+
+    def consents__fetch(self, consent_artefact_id: str):
+        data = {
+            "requestId": str(uuid.uuid4()),
+            "timestamp": datetime.now(tz=timezone.utc).strftime(
+                "%Y-%m-%dT%H:%M:%S.000Z"
+            ),
+            "consentId": consent_artefact_id,
+        }
+
+        return self.request.post(
+            "/v0.5/consents/fetch", data, headers={"X-CM-ID": settings.X_CM_ID}
+        )
+
+    def health_information__cm__request(self, artefact: ConsentArtefact):
+        request_id = str(uuid.uuid4())
+        artefact.consent_id = request_id
+        artefact.save()
+
+        data = {
+            "requestId": request_id,
+            "timestamp": datetime.now(tz=timezone.utc).strftime(
+                "%Y-%m-%dT%H:%M:%S.000Z"
+            ),
+            "hiRequest": {
+                "consent": {"id": str(artefact.artefact_id)},
+                "dataPushUrl": f"https://{settings.BACKEND_DOMAIN}"
```

Comment:
```suggestion
                "dataPushUrl": settings.BACKEND_DOMAIN}
```
this value already has the https prefix